src/current/v22.2/aws-dms.md

```diff
@@ -45,7 +45,11 @@ Complete the following items before starting the DMS migration:
 
     - Drop all [constraints]({% link {{ page.version.version }}/constraints.md %}) per the [AWS DMS best practices](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_BestPractices.html#CHAP_BestPractices.Performance). You can recreate them after the [full load completes](#step-3-verify-the-migration). AWS DMS can create a basic schema, but does not create [indexes]({% link {{ page.version.version }}/indexes.md %}) or constraints such as [foreign keys]({% link {{ page.version.version }}/foreign-key.md %}) and [defaults]({% link {{ page.version.version }}/default-value.md %}).
 
-    - Ensure that any schema changes are also reflected on your tables, or add [transformation rules](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html). If you make substantial schema changes, the AWS DMS migration may fail.
+    - Ensure that any schema changes are also reflected on your target tables, or add [transformation rules](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TableMapping.SelectionTransformation.Transformations.html) to your [table mappings](#step-2-3-table-mappings). If you make substantial schema changes, the AWS DMS migration can fail.
+
+        {% comment %}
```

Comment:
Should this be uncommented? Same question for the other docs

Reply:
Sorry for the confusion @Amruta-Ranade, the comment explaining this was resolved, but here's the thread: https://github.com/cockroachdb/docs/pull/18325#discussion_r1508024865

TLDR this is intentional and the guidance can be uncommented once it is re-tested by the team!

Reply:
Ah, gotcha. Thanks for the clarification!